locale/ja/translation.json

```diff
@@ -108,6 +108,30 @@
     "thumbUp": "はい",
     "thumbDown": "いいえ"
   },
+  "docFeedbackSurvey": {
```

Comment:
Let's align with what @lilin90 and Sai were proposed.

Reply:
Done